docs/cvl/builtin.md

```diff
@@ -106,12 +106,19 @@ rule sanity {
     method f; env e;
     calldataarg arg;
     f(e, arg); 
+    assert true; // for checking "pessimistic" assertions
     satisfy true;
 }
 ```
 
-To find a satisfying trace, the Prover must construct an input
-for which `f` doesn't revert.
+This will create two sub-rules, which will be visible in the report. One
+sub-rule checks the `satisfy true` statement, which is fulfilled if there is an
+input such that `f(e, arg)` runs to completion without reverting. The other
+sub-rule checks the `assert true` statement. Of course, this assertion itself is
+never violated, but the sub-rule contains the "pessimistic" assertions that we
```

Comment:
We should also state what the pessimistic assertions check for - and roughly explain where they are added.

Reply:
hm, the difficulty is that we use the optimistic/pessimistic principle in a whole list of instances (see the CLI options page -- I just checked and saw things I had never heard about 🙃 ) 

One more thing to consider is that I don't want to write something that needs changing whenever one of those options changes..

Best thing I can think of right now is making a glossary entry `pessimistic assertions` (with an alias `optimistic assumptions`) that explains the principle and lists some of the options.. wdyt?

Reply:
alright, did it and re-requested -- what do you think @johspaeth ? (re-requested)